fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBRestrictedIndexQueryTest.java

```diff
@@ -462,4 +462,53 @@ public void queryAllowedUniversalIndex() throws Exception {
         assertFalse(plan2.hasRecordScan(), "should not use record scan");
         assertEquals(-1692774119, plan2.planHash());
     }
+
+    /**
+     * Verify that the query planner uses the specified {@link com.apple.foundationdb.record.query.IndexQueryabilityFilter}.
+     * If both allowed indexes and a queryability filter are set, verify that the planner uses the allowed indexes.
+     */
+    @Test
```

Comment:
I assume that this doesn't work with the new planner (so can't be a `@DualPlannerTest`)?

Reply:
I somehow thought so, but a one-line change in the new planner fixes the issue. This is now a `DualPlannerTest`.